.scripts/build_steps.sh

```diff
@@ -19,7 +19,7 @@ conda-build:
 
 CONDARC
 
-conda install --yes --quiet conda-forge-ci-setup=2 conda-build=3.18.9 -c conda-forge
+conda install --yes --quiet conda-forge-ci-setup=2 conda-build -c conda-forge
```

Comment:
This should be added back to avoid a nasty performance regression when packaging up the install

Reply:
Beat you to it. ;)